### ISSUE TITLE
Clarify + test search updates

### ISSUE THREAD

**Issue by metasj** [prior-art-archive/overview#13]
There is a domain-specific language for searching, written by Cisco in Java to parse query strings into JSON elastic queries (applying filters, priorities, and other sorting functions).  

We've built a jar and deploy it as a lambda trigger whenever anyone searches.
That implementation isn't great, and doesn't work for everything. 
* We've updated to a newer version of Elastic; some things they rely on no longer work the same way  (special character + filter processing)
* The previous Elastic server config was updated manually through the web UI; no config file.  We may need to have a conf call w/ Cisco to extract what config details they used / we need.  (Joel has a running gist of config issues, we should do sth repeatable here rather than doing it by hand ourselves)

A few related aspects of v1 PAA architecture is still running on their machines, including Kafka queues.  And one aspect involves round-trips into Google  (generation of CPC codes, which are then added to our database and made searchable)

**Comment by metasj** [comment]
Related: Make CPC searches work.  #4

**Comment by reefdog** [comment]
@metasj Did CPC searches work before migration?

**Comment by metasj** [comment]
Yes.

**Comment by metasj** [comment]
Joel -- can you summarize this + break out remaining issues?
